packages/scripts/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@wordpress/scripts",
+  "version": "1.0.0",
+  "description": "Collection of JS scripts for WordPress development",
+  "author": "WordPress",
+  "license": "GPL-2.0+",
+  "keywords": [
+    "scripts",
+    "reusable-scripts"
+  ],
+  "homepage": "https://github.com/WordPress/packages/tree/master/packages/scripts/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/packages.git"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/packages/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=6"
+  },
+  "files": [
+    "bin"
+  ],
+  "bin": {
+    "wp-scripts": "./bin/wp-scripts.js"
+  },
+  "dependencies": {
+    "cross-spawn": "5.1.0",
+    "grunt": "1.0.1",
+    "grunt-wp-i18n": "1.0.1",
+    "grunt-wp-readme-to-markdown": "2.0.1"
```

Comment:
@stephenharris is the maintainer. 

In general, I agree that lessening the dependency on Grunt makes sense, but at the same time it's not exposed to the plugin developer, so it is just an implementation detail rather than an api detail.